interactions/api/gateway.py

```diff
@@ -320,6 +320,8 @@ def check_sub_auto(option: dict) -> tuple:
                                     )
                     elif data["type"] == InteractionType.MESSAGE_COMPONENT:
                         _name = f"component_{context.data.custom_id}"
+                        if context.data._json.get("values"):
+                            _args.append(context.data.values)
```

Comment:
Because in the callback function it will be much easier to immediately receive the selected value than to request it from nowhere (I, for example, did not immediately understand that the value can be obtained from the context).